src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Figure {
+    private double area;
+    private String color;
+
+    public double getArea() {
+        return area;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void getDraw() {
+    }
```

Comment:
```suggestion
public abstract class Figure implements Drawable, AreaCalculator{
    private double area;
    private String color;

    public double getArea() {
        return area;
    }
    
    public void setArea(double area) {
        this.area = area;;
    }

    public String getColor() {
        return color;
    }
    
    public void setColor(String color) {
    this.color = color;
    }
]
```

Нам нужно создать 2 интерфейса Drawable и AreaCalculator с методами calculateArea() и draw(). Твой предыдущий интерфейс  `Forms` объеденял в себе логику этих двух интерфейсов (Drawable и AreaCalculator ), но он нам не нужен, нам нужны непосредственно Drawable и AreaCalculator